CMakePresets.json

```diff
@@ -6,8 +6,6 @@
       "hidden": true,
       "generator": "Ninja",
       "cacheVariables": {
-        "Languages": "CHINESE;DUTCH;ENGLISH;FRENCH;GERMAN;ITALIAN;POLISH;RUSSIAN",
```

Comment:
Having them listed here allows for quicker modification to a custom use case

Reply:
yea, sure